acquisitions/src/main/resources/thunderjet/mod-orders/features/find-holdings-by-location-and-instance-for-mixed-pol-with-create-inventory-none.feature

```diff
@@ -0,0 +1,166 @@
+@parallel=false
+# for https://issues.folio.org/browse/MODORDERS-871
+Feature: find-holdings-by-location-and-instance-for-mixed-pol
+
+  Background:
+    * url baseUrl
+#    * callonce dev {tenant: 'testorders'}
+    * callonce loginAdmin testAdmin
+    * def okapitokenAdmin = okapitoken
+    * callonce loginRegularUser testUser
+    * def okapitokenUser = okapitoken
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json'  }
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*'  }
+    * configure headers = headersUser
+
+    * callonce variables
+
+    * def fundId = callonce uuid1
+    * def budgetId = callonce uuid2
+    * def orderId1 = callonce uuid3
+    * def orderId2 = callonce uuid4
+    * def poLineId1 = callonce uuid5
+    * def poLineId2 = callonce uuid6
+
+  Scenario: Create finances
+    * configure headers = headersAdmin
+    * call createFund { 'id': '#(fundId)'}
+    * call createBudget { 'id': '#(budgetId)', 'allocated': 10000, 'fundId': '#(fundId)'}
+
+  Scenario: Create first order
```

Comment:
For simple scanarious you can use already written functions to create/open orders like this as example:

Scenario: Create composite order
    * def v = call createOrder { id: #(orderId) }

  Scenario: Create order line
    * def v = callonce createOrderLine { id: #(poLineId), orderId: #(orderId), fundId: #(globalFundId) }

  Scenario: Open order
    * def v = callonce openOrder { orderId: "#(orderId)" }
    
These functions located in reusable package: https://github.com/folio-org/folio-integration-tests/tree/master/acquisitions/src/main/resources/thunderjet/mod-orders/reusable

Reply:
@SerhiiNosko :

Yes , i have seen those, create orderLine is  reading different file and particulars are also different , thats why i written. 
if i  want to reuse, i need to write seperate feature file. 

let me know if  i can do this . thanks

Reply:
I see, okay lets use as is, no changes required

Reply:
fixed.